taxi/urls.py

```diff
@@ -13,39 +12,43 @@
     ManufacturerCreateView,
     ManufacturerUpdateView,
     ManufacturerDeleteView,
+    DriverCreateView,
+    DriverDeleteView,
+    DriverLicenseUpdateView,
+    assign_driver_to_car,
 )
 
 urlpatterns = [
     path("", index, name="index"),
-    path(
-        "manufacturers/",
-        ManufacturerListView.as_view(),
-        name="manufacturer-list",
-    ),
-    path(
-        "manufacturers/create/",
-        ManufacturerCreateView.as_view(),
-        name="manufacturer-create",
-    ),
-    path(
-        "manufacturers/<int:pk>/update/",
-        ManufacturerUpdateView.as_view(),
-        name="manufacturer-update",
-    ),
-    path(
-        "manufacturers/<int:pk>/delete/",
-        ManufacturerDeleteView.as_view(),
-        name="manufacturer-delete",
-    ),
+    path("manufacturers/", ManufacturerListView.as_view(),
+         name="manufacturer-list"),
+    path("manufacturers/create/", ManufacturerCreateView.as_view(),
+         name="manufacturer-create"),
+    path("manufacturers/<int:pk>/update/", ManufacturerUpdateView.as_view(),
+         name="manufacturer-update"),
+    path("manufacturers/<int:pk>/delete/", ManufacturerDeleteView.as_view(),
+         name="manufacturer-delete"),
+    path("drivers/", DriverListView.as_view(),
+         name="driver-list"),
+    path("drivers/<int:pk>/", DriverDetailView.as_view(),
+         name="driver-detail"),
+    path("drivers/create/", DriverCreateView.as_view(),
+         name="driver-create"),
+    path("drivers/<int:pk>/update_license/", DriverLicenseUpdateView.as_view(),
+         name="driver-update"),
```

Comment:
The URL name 'driver-update' should use a hyphen instead of an underscore according to the checklist requirement: 'Use `-` to split words in URL identification parameter `name`, not the `_`.'